default/data/ui/views/rw_exec.xml

```diff
@@ -48,8 +48,10 @@
       <single>
         <title>Active Zoom Meetings</title>
         <search>
-          <query>`rw_vc_indexes` sourcetype="zoom:webhook" TERM(meeting.started) OR TERM(meeting.ended) event="meeting.started" OR event="webinar.started"
-| timechart partial=f span=1h dc(payload.object.uuid) as active_meeting</query>
+          <query>`rw_vc_indexes` sourcetype="zoom:webhook" TERM(meeting.started) OR TERM(meeting.ended) event="meeting.started" OR event="webinar.started" OR event="meeting.ended" OR event="webinar.ended" 
+| stats latest(event) AS event latest(_time) AS _time BY payload.object.uuid 
+| where event="meeting.started" 
```

Comment:
Change from `| where event="meeting.started"` to `| where event="meeting.started" OR event="webinar.started"` to account for webinars.